### ISSUE TITLE
Stakeholder Meeting

### ISSUE THREAD

**Issue by fraxinus1098** [yale-research-job-board/.github#14]

**Comment by fraxinus1098** [comment]
@yashswijain what is the status update on this? When reaching out, please CC me, Kyle, and whoever signs up for this task

**Comment by fraxinus1098** [comment]
We should also come up with a list of questions we want to ask

**Comment by fraxinus1098** [comment]
@yashswijain has sent out the email to tristen and is pending response. Follow up Monday 12/23? Ideally try to interview him next week

**Comment by yiyanghe1** [comment]
Tristan replied last night asking for availability the week of 1/6, which is a bit late for us. We should push to see if he can chat later this week after Xmas or early next week before New Year's. Alternatively, we can draft up a set of questions to get more details about the product and let him respond over email. We can also ask Professor Jensen for a nudge if that would help expedite the process. @yashswijain and @komalbagrodia97 let me know if yall have other thoughts

**Comment by komalbagrodia97** [comment]
@yiyanghe1 Agreed. I am working on the questions that we can ask him to understand his requirements.

**Comment by komalbagrodia97** [comment]
@yashswijain @yiyanghe1 Feel free to modify the list below:

## **Vision & Business Goals**

**Core Questions:** *To establish the fundamental purpose and market positioning*

* What specific problem are you trying to solve?  
* Do you want any unique value proposition?  
* Who do you consider your main competitors and why?

  ## **Target Audience Definition**

  **Primary Focus:** *To determine the core user base and feature prioritisation*  
* Which user group is your priority \- employers, job seekers, or both?  
* What specific industry sectors or job categories are you targeting?

  ## **MVP Feature Requirements**

  **Essential Elements:** *To define a minimum scope for a viable product*  
* What are your non-negotiable features for launch?  
* Describe your ideal user journey from start to finish  
* Are any third-party integrations essential for launch?

  ## **Project Parameters**

  **Timeline and Success Metrics:** *To establish clear deliverables and measurement criteria*  
* When do you need the MVP launched?  
* How will you measure the MVP's success?

**Comment by yashswijain** [comment]
I like the list. If everyone is okay, I can send it over email to the Prof. to get an initial response given the time crunch.

**Comment by fraxinus1098** [comment]
My only suggestion is to fill in some of the questions with what we already know and some assumptions based on the first meeting with Kyle. A lot of times, users do not know what they exactly want, so you have to guide them a bit. I recommend sending it to him this week since next week is New Years.

@yashswijain 

CC @komalbagrodia97 @yiyanghe1

**Comment by yiyanghe1** [comment]
That list looks good to me as well. One suggestion - it might be worth asking Tristan to share links or screenshots to examples of job boards features he likes, so we have some visual inspiration to work with as we start building the product
